<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.algaworks.algashop.ordering.domain.entity.CustomerTest" tests="6" skipped="0" failures="0" errors="0" timestamp="2025-07-13T13:55:21.759Z" hostname="efratec" time="0.086">
  <properties/>
  <testcase name="given_invalidEmail_whenTryUpdatedCustomerEmail_shouldGenerateException()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.055"/>
  <testcase name="given_brandNewCustomer_whenAddInvalidLoyaltyPoints_shouldGenerateException()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.001"/>
  <testcase name="given_unarchivedCustomer_whenArchive_shouldAnonymize()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.016"/>
  <testcase name="given_archivedCustomer_whenTryToUpdate_shouldGenerateException()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.002"/>
  <testcase name="given_brandNewCustomer_whenAddLoyaltyPoints_shouldSumPoints()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.003"/>
  <testcase name="given_invalidEmail_whenTryCreateCustomer_shouldGenerateException()" classname="com.algaworks.algashop.ordering.domain.entity.CustomerTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
